value/value.md

```diff
@@ -231,87 +225,180 @@ the player to your app, and for which nothing already exists, add your own.
 
 ### Payment calculation
 
-The interval payment calculation is:
+A given recipient will receive
+```
+(Payment amount) * (Interval payout) * (Interval count)
```

Comment:
Can payment amount, interval payout, and interval count be defined here? I assume payment amount is the user's percentage of the total payment amount and interval payout * count is the total payment amount.

Reply:
Good catch! That part should be adjusted.

Reply:
What do you think?

https://github.com/Podcastindex-org/podcast-namespace/pull/596/commits/87284605b6dfe2c7446d027748db5887e661b585

Reply:
Is it worth making a distinction between intervaled and non-intervaled payments? Is the per-recipient calculation different for them?

Reply:
The calculation is not different, it's just that `Interval count` may not make sense for boosts, so I thought it needs to be at least acknowledged.

I'm not really sure about that part. Open to suggestions on how to better word it. Or if it's necessary at all.

Reply:
@MerryOscar I agree the `split` shouldn't be a percentage, but should have the percentage derived from the `split total`. 
However, what we need a consensus on is how to calculate the `split total`, because some may add all splits including `fee=true` and others may use just the `fee=false` splits. I'm in favor of simply adding all of the splits including 'fee=true' to come up with the total split, but that's not a hill I'm willing to die on. 

My reason for adding all of the splits is it prevents this scenario.  Imagine the 'fee=false' are `a: 50` and `b: 50`. So, the `split total` so far is 100. Now, imagine someone puts in their podcast a 'fee=true' and the split is 200. How is that handled for a 1000 sat boost? By adding all of the splits, including `fee=true`, this becomes less problematic.

A hill I am willing to die on is I won't ever **add** `fees` to the boost. When I boost 1000 sats, I expect 1000 sats to be taken out of my wallet, and would be bothered if I saw 1200 sats removed because the podcaster had a bunch of services adding up to 20%. That 20% should come out of the podcasters pocket not mine. And in the case above, where fees add up to 200%, the podcaster would end up sending 3000 sats even though they only intended to send 1000 sats.

The only caveat to that would be if the app decides to take a cut. I could see an argument for adding the app's cut to the top of the boost, almost like a tax, since the podcaster didn't decide which app the listener would use, and one app could decide to take a 20% cut if they wanted to. In that case, the app shouldn't get 20% of what was intended for the podcaster, because the podcaster didn't have a say in that. However, with the splits, the podcaster has complete control of the split amounts, and should be responsible for paying them out of the listeners initial boost amount.